cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -154,10 +165,24 @@ protected DatasetClient getDatasetClient() {
     return new DatasetClient(getClientConfig());
   }
 
+  protected Id.Namespace createNamespace(String name) throws Exception {
+    Id.Namespace namespace = new Id.Namespace (name);
+    createNamespace(new NamespaceMeta.Builder().setName(namespace).build());
+    return namespace;
+  }
+
+  protected void createNamespace(NamespaceMeta namespaceMeta) throws Exception {
```

Comment:
Instead of implementing these `createNamespace` and `deleteNamespace` method, I think its fine to simply expose them via the `TestManager`.
So in the test case, user can simply call `getTestManager().deleteNamespace`.

I think that `TestManager` class has so many methods, that having the equivalent (wrapper) methods in `IntegrationTestBase.java` would be redundant and messy.